tests/singlenode_key_ed25519/singlenode_test.go

```diff
@@ -58,18 +57,20 @@ func TestSingleNodeDeployment(t *testing.T) {
 	assert.NotEmpty(t, wgConfig)
 
 	pIP := strings.Split(publicIP, "/")[0]
```

Comment:
use pIP should be publicIP, rename the old publicIP variable to publicIPWithCidr or something